cmd/algorelay/eb/eb.go

```diff
@@ -19,7 +19,7 @@ package eb
 // Relay represents the configuration data necessary for a single Relay
 type Relay struct {
 	ID             int64 // db key injected when loaded
-	IPOrDNSName    string
+	Address        string
```

Comment:
Address is an overloaded term in our system, could you add `// IP / DNS name` here?

Reply:
done.